ros/ubuntu/orocos_toolchain.repos

```diff
@@ -0,0 +1,5 @@
+repositories:
+  orocos_toolchain:
+    type: git
+    url: https://github.com/orocos-toolchain/orocos_toolchain.git
+    version: ros2
```

Comment:
This is the version for ros2, I guess:
```suggestion
   version: toolchain-2.9

```

Reply:
Actually, the branch `toolchain-2.9` contains a bug that prevents from building when using `focal` and `noetic`, referred to in [this issue](https://github.com/orocos-toolchain/typelib/issues/130).

Reply:
For now it was intentional to use the `ros2` branch. It is actually not specific to ROS 2, but has patches that have been applied while working on the ROS 2 integration and that still need to be submitted as individual pull requests to `master`. It also disables the build of other toolchain packages than RTT and OCL, for exactly the reasons you mentioned.

Once that is done and all required patches and also branch `toolchain-2.9` itself has been merged back into `master`, we can switch back to the default branch `HEAD` here, too.